server/models/handlers/school.js

```diff
@@ -0,0 +1,8 @@
+// Create stub which returns an array of Worker ID's given an input of school ID or school Name
+async function getWorkerIdsForSchool(schoolId) {
+      var workerIds = [8000000, 8000001, 8000002, 8000003]
```

Comment:
Hi Simran,
There's a missing semicolon on this line after the ]. I would recommend inserting that semicolon to prevent runtime issues.

Reply:
Hi Amy! Thank you this issue has been addressed!